pkg/products/threescale/reconciler.go

```diff
@@ -2023,13 +2021,15 @@ func (r *Reconciler) reconcileBlackboxTargets(ctx context.Context, client k8scli
 		}
 
 		// Create a blackbox target for the developer console ui
-		threescaleRoute, err := r.getThreescaleRoute(ctx, client, "system-developer", func(r routev1.Route) bool {
-			return strings.HasPrefix(r.Spec.Host, "3scale.")
-		})
+		threescaleRoute, err := r.getThreescaleRoute(ctx, client, "system-developer", nil)
```

Comment:
Has this been tested on a running cluster - that no UIBBT alerts are firing? I was under the impression that this would return 2 routes?